src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -0,0 +1,79 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the class Wedge3D.
+
+#pragma once
+
+#include <array>
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Direction.hpp"
+
+namespace CoordinateMaps {
+
+/*!
+ * \ingroup CoordinateMaps
+ *
+ *  \brief Three dimensional map from the cube to a wedge.
+ *
+ *  \detail The wedge is shaped such that six of them put together
```

Comment:
The wedge is constructed by linearly interpolating between a face of a cube and a face of a cubed sphere, such that six wedges put together make up an object where one of the surfaces is spherical and the other surface is cubical.
The values of `cube_radius` (which is the length from the origin to a corner) and `sphere_radius` determine which surfaces are the interior and exterior. 

The `sphericity` of the spherical surface can also be adjusted to values between zero and one, where a value of 1 corresponds to a spherical surface and a value of zero corresponds to the surface of a cube. This allows for the construction of a `Domain` where one of the surfaces is that of a bulged cube.

Reply:
Also please document that the radial coordinate is the third logical coordinate.